fms_fsdp/utils/train_utils.py

```diff
@@ -66,16 +90,22 @@ def train(
                 (batch_idx - start_step) * world_size * cfg.batch_size * cfg.seq_length
             )
             if rank == 0:
+                seen_token = n_tok + elapsed_tokens
```

Comment:
nit: n_tok is a bit confusing here, in a separate PR we might want to re-name it to better reflect what it means.

Reply:
I think this is a good chance to make this nit change.